configuration.rst

```diff
@@ -39,6 +39,8 @@ db-anon-role             String             Y
 db-pool                  Int     10
 db-pool-timeout          Int     10
 db-extra-search-path     String  public
+db-channel               String  pgrst
+db-channel-enabled       Boolean False
```

Comment:
@laurenceisla Now that I remember, we agreed somewhere that `db-channel-enabled` must be True by default. We wanted to be conservative because of pgbouncer, but we already have `db-prepared-statements` as True, which wouldn't work with pgbouncer anyway.

You could make this change in postgrest itself - change `db-channel-enabled` to True by default.

Reply:
Found the comment where you agreed to it  https://github.com/PostgREST/postgrest/issues/1622#issuecomment-733066644 and Wolfgang gives a very good explanation on to why it should be True by default. I'll make a PR in core and come back to the docs when the changes are merged.